src/worker.cpp

```diff
@@ -1267,6 +1267,17 @@ void Worker::collectFrusFromJson()
             continue;
         }
 
+#ifdef IBM_SYSTEM
```

Comment:
I knew we discussed this, but I was giving it a second thought, don;t see a valid reason why even "isChassisPowerOn" should be under conditional compilation.
I would suggest get rid of the tags here and please create an issue ta take that API out of conditional compilation and into a utility file.

Reply:
If I remove the tags here, I would need to move `isChassisPowerOn()` out of conditional compilation in this PR itself, otherwise CI job would fail for this PR.

Currently on the Jenkins CI job, `ibm_system` option is set to `disabled`. 
`/home/ibm/openbmc-dev-jenkins/workspace/OpenBMC-Dev/build/github-public/repo-ci/public-ibm-openbmc-downstream-repo-ci-pr-openpower-vpd-parser/ibm-openbmc/openpower-vpd-parser > meson setup --reconfigure build -Db_colorout=never -Dwerror=true -Dwarning_level=3 -Dcpp_args='-DBOOST_USE_VALGRIND' -Ddebug=true -Doptimization=g -Dtests=enabled`

Reply:
Created issue https://github.com/ibm-openbmc/openpower-vpd-parser/issues/436 
I think `isChassisPowerOn()` should be moved to `vpd_specific_utility.hpp`. Is that fine @SunnySrivastava1984 ?

Reply:
This is addressed. API `isChassisPowerOn()` is moved out of conditional compilation by https://github.com/ibm-openbmc/openpower-vpd-parser/pull/440

Reply:
ok